deepcell/datasets/__init__.py

```diff
@@ -42,29 +45,20 @@ class Dataset:
             (relative to ~/.keras/datasets).
         url (str): URL of dataset in S3.
         file_hash (str): md5hash for checking validity of cached file.
-        metadata (dict): miscellaneous other data for dataset
+        secure (bool): True if the dataset requires a deepcell api key for download. Default False
     """
 
     def __init__(self,
                  path,
                  url,
                  file_hash,
-                 metadata):
+                 secure=False):
         self.path = path
         self.url = url
+        self.secure = secure
         self.file_hash = file_hash
-        self.metadata = metadata
 
-    def _load_data(self, path, mode, test_size=0.2, seed=0):
-        """Loads dataset.
-
-        Args:
-            test_size (float): fraction of data to reserve as test data
-            seed (int): the seed for randomly shuffling the dataset
-
-        Returns:
-            tuple: (x_train, y_train), (x_test, y_test).
-        """
+    def _get_data(self):
         basepath = os.path.expanduser(os.path.join('~', '.keras', 'datasets'))
```

Comment:
We could consider changing the base download to `~/.deepcell/datasets` and `~/.deepcell/models`